etl/definitions.py

```diff
@@ -14,7 +14,7 @@
 )
 
 openai_settings = OpenAiSettings(
-    openai_api_key=EnvVar("WIKIPEDIA_TRANSFORM_OPENAI_API_KEY").get_value("")
+    openai_api_key=EnvVar("ETL_OPENAI_API_KEY").get_value("")
```

Comment:
Nit: I'd just name this OPENAI_API_KEY to go along with all the libraries. No need to prefix it.

Reply:
That makes sense. I have changed the name.